main.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-alert */
+/* eslint-disable comma-dangle */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] Please do not disable linters. Linters help programmers to write code of the same quality and standard. They promote good coding practice 🔖